src/main/java/ch/njol/skript/events/EvtEntityBlockChange.java

```diff
@@ -31,12 +32,12 @@
 import org.bukkit.entity.Silverfish;
 import org.bukkit.event.Event;
 import org.bukkit.event.entity.EntityChangeBlockEvent;
-import org.eclipse.jdt.annotation.Nullable;
+import org.jetbrains.annotations.Nullable;
 
 import java.util.Locale;
 
 public class EvtEntityBlockChange extends SkriptEvent {
-	
+
 	static {
 		Skript.registerEvent("Enderman/Sheep/Silverfish/Falling Block", EvtEntityBlockChange.class, EntityChangeBlockEvent.class, ChangeEvent.patterns)
 				.description(
```

Comment:
This event would benefit from telling users that `event-block` is old and `event-blockdata` is new, since there's no real way to figure this out other than javadocs/source code/trial and error